crates/holochain/tests/tests/gossip/mod.rs

```diff
@@ -14,12 +15,20 @@ use holochain_zome_types::record::Record;
 
 /// Test that conductors with arcs clamped to zero do not gossip.
 #[tokio::test(flavor = "multi_thread")]
-#[cfg_attr(target_os = "macos", ignore = "flaky")]
 async fn get_with_zero_arc_2_way() {
     holochain_trace::test_run();
 
     // Standard config with arc clamped to zero and publishing off
-    let mut conductors = SweetConductorBatch::from_standard_config_rendezvous(2).await;
+    let empty_arc_conductor_config = SweetConductorConfig::rendezvous(false)
```

Comment:
This test wasn't actually setting the target arc to empty. For this test it didn't result in a difference, but formally it's more correct now.